tests/foreman/api/test_location.py

```diff
@@ -0,0 +1,210 @@
+"""Unit tests for the ``locations`` paths.
+
+A full API reference for locations can be found here:
+http://theforeman.org/api/apidoc/v2/locations.html
+
+"""
+from ddt import ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from random import randint
+from robottelo.common.decorators import data
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=R0904
```

Comment:
Current versions of Pylint know to ignore all the methods inherited from unittest, or something like that. As a result, you can just drop this line.

By the way, an even nicer way to write this is:

    # pylint:disable=too-many-public-methods